### PR TITLE
Add the capacity for tile descriptions

### LINKED ISSUE #8595: Tiles should be able to provide a description like objects and items.
### Description

tiles currently can provide names, but cannot provide any further information when examined. It would be useful to be able to provide descriptions about the nature of the tile for educating players and for flavor.

### PR DESCRIPTION
**Pull request by NoooneyDude**
CL: [New] Added the ability to give base descriptions to tiles for player examination.
CL: [Improvement] Gave the regular window tile a description.
- Namespaces a few classes, particularly tile classes.
closes #8595

### CODE REVIEW

File: UnityProject/Assets/Scripts/Tilemaps/Editor/Brushes/LevelBrushEditor.cs [hunk 1]
```diff
@@ -1,11 +1,12 @@
 using System.Linq;
 using TileManagement;
+using Tiles;
 using UnityEditor;
 using UnityEngine;
 using UnityEngine.Tilemaps;
 
 
-	[CustomEditor(typeof(LevelBrush))]
+[CustomEditor(typeof(LevelBrush))]
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Tilemaps/Editor/Brushes/LevelBrushEditor.cs:9]: o-O

Comment (NoooneyDude) [reply UnityProject/Assets/Scripts/Tilemaps/Editor/Brushes/LevelBrushEditor.cs:9]: I'm so sorry, it's the editor's fault! : )

Comment (NoooneyDude) [reply UnityProject/Assets/Scripts/Tilemaps/Editor/Brushes/LevelBrushEditor.cs:9]: Just not enough to fix 'em all though 😉

**Comment by PerfectTangent** [comment]
praise tile description capacitance.

**Review by corp-0 — APPROVED** [review]

**Pull request merged**